### DISCUSSION TITLE
I can not find my API token

### DISCUSSION THREAD

**Discussion by Bagency** [codemagic-ci-cd/codemagic-docs#1105] [Q&A] [answered]
Hi dears;
Hope you are doing well

I am new here and I found that to get my API token , I have to go user settings

But I found only User preferences , and there is no API token to show

Please advice
Thanks in advance

**Comment by mikhail-tokarev** [comment, marked as answer]
Hi @Bagency 
all integration settings were moved to Teams > Personal Account
